client/packages/common/src/ui/layout/tables/columns/PaginationRow/PaginationRow.test.tsx

```diff
@@ -6,69 +6,69 @@ import { TestingProvider } from '../../../../../utils';
-          <PaginationRow
-            page={0}
-            offset={offset}
-            total={total}
-            first={first}
-            onChange={jest.fn()}
-          />
-        </TableProvider>
-      </TestingProvider>
-    );
-
-    const node1 = getByText(/showing/i);
-    const node2 = getByText(`${offset + 1}-${first + offset}`);
-    const node3 = getByText(/of/i);
-    const node4 = getByText(`${total}`);
-
-    expect(node1).toBeInTheDocument();
-    expect(node2).toBeInTheDocument();
-    expect(node3).toBeInTheDocument();
-    expect(node4).toBeInTheDocument();
-  });
-
-  it('Renders at least one page button', () => {
-    const { getByRole } = render(
-      <TestingProvider>
-        <TableProvider
-          createStore={createTableStore}
-          queryParamsStore={createQueryParamsStore({
-            initialSortBy: { key: 'id' },
-          })}
-        >
-          <PaginationRow
-            page={0}
-            offset={1}
-            total={1}
-            first={1}
-            onChange={jest.fn()}
-          />
-        </TableProvider>
-      </TestingProvider>
-    );
-
-    const node = getByRole('button', { name: 'page 1' });
-
-    expect(node).toBeInTheDocument();
-  });
+  // it('Renders a string : Showing X-Y of Z', () => {
+  //   const offset = 0;
+  //   const first = 20;
+  //   const total = 40;
+
+  //   const { getByText } = render(
+  //     <TestingProvider>
+  //       <TableProvider
+  //         createStore={createTableStore}
+  //         queryParamsStore={createQueryParamsStore({
+  //           initialSortBy: { key: 'id' },
+  //         })}
+  //       >
+  //         <PaginationRow
+  //           page={0}
+  //           offset={offset}
+  //           total={total}
+  //           first={first}
+  //           onChange={jest.fn()}
+  //         />
+  //       </TableProvider>
+  //     </TestingProvider>
+  //   );
+
+  //   const node1 = getByText(/showing/i);
+  //   const node2 = getByText(`${offset + 1}-${first + offset}`);
+  //   const node3 = getByText(/of/i);
+  //   const node4 = getByText(`${total}`);
+
+  //   expect(node1).toBeInTheDocument();
+  //   expect(node2).toBeInTheDocument();
+  //   expect(node3).toBeInTheDocument();
+  //   expect(node4).toBeInTheDocument();
+  // });
+
+  // it('Renders at least one page button', () => {
+  //   const { getByRole } = render(
+  //     <TestingProvider>
+  //       <TableProvider
+  //         createStore={createTableStore}
+  //         queryParamsStore={createQueryParamsStore({
+  //           initialSortBy: { key: 'id' },
+  //         })}
+  //       >
+  //         <PaginationRow
+  //           page={0}
+  //           offset={1}
+  //           total={1}
+  //           first={1}
+  //           onChange={jest.fn()}
+  //         />
+  //       </TableProvider>
+  //     </TestingProvider>
+  //   );
+
+  //   const node = getByRole('button', { name: 'page 1' });
+
+  //   expect(node).toBeInTheDocument();
+  // });
```

Comment:
plan to move it back in? or remove entirely?

Reply:
oops. that was while fixing the pagination test. good spotting!

Reply:
and for the tests.. no. When I upgraded the first time I tried this https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#configuring-your-testing-environment but it didn't help.

Reply:
no, wait. good catch! In removing the warning, I've found a couple of tests that were giving a false positive. 
Have fixed up and the suite is looking sweet!